app/pb_kits/playbook/pb_icon_title/_icon_title.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= content_tag(:div, id: object.id,  data: object.data, class: object.classname(object.kit_class)) do %>
+
+<%= pb_rails("icon_circle", props: {
+  icon: "#{object.icon}",
+  size: object.column_sizing ,
+  variant:"#{object.icon_variant}"}) %>
+<div class="content">
+
+  <%= pb_rails("title", props: { text: "Value", size: object.size === "lg" ? 1 : 3 , dark: object.dark === "dark" ?  true : false }) %>
+  <%= pb_rails("caption", props: { text: "Description" , dark: object.dark === "dark" ?  true : false}) %>
```

Comment:
Since we are doing logic on prop values for `Title` and `Caption`, this should be moved to icon_title.rb.  Please see [Label Value](https://github.com/powerhome/playbook/blob/master/app/pb_kits/playbook/pb_label_value/label_value.rb#L26) kit for reference.

Reply:
@kre8sions To help on this, I thought I would help explain this a bit more!  

By bringing in the title and caption nested kits into the icon_title.rb file, you can do all the logic within your kit class:
```ruby
def value
  if is_set? configured_value
    title_value = default_value(configured_value, "Value")
    size_value = size === "lg" ? 1 : 3
    dark_value = dark === "dark" ?  true : false
    
    pb_value = Playbook::PbTitle::Title.new(text: title_value, size: size_value, dark: dark_value)
    ApplicationController.renderer.render(partial: pb_value, as: :object)
  end
end

def description 
  dark_value = dark === "dark" ?  true : false
  
  pb_description = Playbook::PbCaption::Caption.new(text: description, dark: dark_value)
  ApplicationController.renderer.render(partial: pb_description, as: :object)
end
```

And then your kit partial (_icon_title.html.erb) would look like the following:
```erb
<%= content_tag(:div, 
    id: object.id,  
    data: object.data, 
    class: object.classname(object.kit_class)) do %>

  <%= pb_rails("icon_circle", props: {
    icon: "#{object.icon}",
    size: object.column_sizing ,
    variant:"#{object.icon_variant}"}) %>
  <div class="content">
    <%= object.value %>
    <%= object.description %>
  </div>
  
<% end %>
```